<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvailableSlots</class>
 <widget class="QFrame" name="AvailableSlots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>641</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1032</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,255);
border-top-right-radius:80px;
border-top-left-radius:80px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>1032</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(72,170,173,1);
border-bottom-left-radius:80px;
border-bottom-right-radius:80px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>140</y>
      <width>301</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
border-bottom-right-radius:80px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Available Slots</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Appointment">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>210</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_Appointment{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 256));
    color:rgba(255,255,255,210);
    border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>Appointment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Cancelation">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>340</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_Cancelation{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 256));
    color:rgba(255,255,255,210);
    border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>Cancelation</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);


</string>
    </property>
    <property name="text">
     <string>6.00pm - 6.20pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>6.20pm - 6.40pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>6.40pm -7.00pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>7.00pm - 7.20pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>7.20pm - 7.40pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>7.40pm - 8.00pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>8.00pm - 8.20pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>8.20pm - 8.40pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot9">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>510</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>8.40pm - 9.00pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot10">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>132</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>9.00pm - 9.20pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot11">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>390</y>
      <width>132</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>9.20pm - 9.40pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_slot12">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>510</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,125);
</string>
    </property>
    <property name="text">
     <string>9.40pm -10.00pm</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_doctor">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>171</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255,0,0,1);</string>
    </property>
    <property name="text">
     <string>FIRST CARE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>81</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Family Medical Service</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_check_slots">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_check_slots{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 256));
    color:rgba(255,255,255,210);
    border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>Check for available slots</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton_Appointment</zorder>
   <zorder>pushButton_Cancelation</zorder>
   <zorder>t_slot1</zorder>
   <zorder>t_slot2</zorder>
   <zorder>t_slot3</zorder>
   <zorder>t_slot4</zorder>
   <zorder>t_slot5</zorder>
   <zorder>t_slot6</zorder>
   <zorder>t_slot7</zorder>
   <zorder>t_slot8</zorder>
   <zorder>t_slot9</zorder>
   <zorder>t_slot10</zorder>
   <zorder>t_slot11</zorder>
   <zorder>t_slot12</zorder>
   <zorder>comboBox_doctor</zorder>
   <zorder>label_17</zorder>
   <zorder>label_18</zorder>
   <zorder>label_19</zorder>
   <zorder>pushButton_check_slots</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
